common/rpctypes/types.go

```diff
@@ -17,9 +17,6 @@ import (
 
 // JSON RPC method names that we serve on the localhost server
 const (
-	NetDiscover         = "net_discover"
-	NetQueryPeer        = "net_queryPeer"
-	SubscribeNewPeer    = "net_subscribeNewPeer"
 	SubscribeMakeOffer  = "net_makeOfferAndSubscribe"
 	SubscribeTakeOffer  = "net_takeOfferAndSubscribe"
```

Comment:
If we want to maintain/test less code, I don't see how these 2 methods are any different than calling their `http` equivalent and then immediately calling `swap_subscribeStatus`. There is no race condition. If a status is added to the channel between the http and ws requests, it will still be waiting for the user when the 2nd request starts.

Reply:
yeah, we can probably remove these in favour of the above, feel free to open an issue for it

Reply:
I'll just leave it for now. It is not as simple as I though to combine them when I wrote that message. In its current implementation, `swap_subscribeStatus` assumes the swap is currently ongoing, but that is not the case when the maker listens for state changes on the offer he just made.